psh/test-autocompletion.py

```diff
@@ -47,12 +47,14 @@ def assert_hints(p, path, hints):
     for h in hints:
         p.expect_exact(h)
     p.send(ENTER)
-    assert_prompt(p, 'No prompt after hints test', timeout=1)
+    psh.assert_prompt(msg='No prompt after hints test', timeout=1)
 
 
 def harness(p):
-    run_psh(p)
-    assert_only_prompt(p)
+    global psh
```

Comment:
I thought, that is more convenient to use instead of adding arguments to all, or most of the functions. Please let me know if it's really better to change it, I will do it then

Reply:
In the other tests we don't push `p` object to the global scope.

Reply:
As I said, I will change it then